### PR TITLE
Test person predicates

### PR DESCRIPTION
**Pull request by marcusczh**

### CODE REVIEW

File: src/test/java/longtimenosee/model/person/predicate/RiskAppetiteMatchesInputPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        String firstPredicateInput = "H";
+        String secondPredicateInput = "M";
+
+        RiskAppetiteMatchesInputPredicate firstPredicate = new RiskAppetiteMatchesInputPredicate(firstPredicateInput);
+        RiskAppetiteMatchesInputPredicate secondPredicate = new RiskAppetiteMatchesInputPredicate(secondPredicateInput);
+
+        // EP: null value
+        assertFalse(firstPredicate.equals(null)); // Boundary value
+
+        // EP: same object
```
Comment (cowlinn) [inline src/test/java/longtimenosee/model/person/predicate/RiskAppetiteMatchesInputPredicateTest.java:22]: In general, very good use of Equivalence Partitions! Good to see the team putting into practice what we learned during tutorials.

File: src/test/java/longtimenosee/logic/parser/FindCommandParserTest.java [hunk 2]
```diff
@@ -71,17 +71,17 @@ public void parse_invalidNameArgs_throwsParseException() {
     public void parse_validPhoneArgs_returnsFindCommand() {
         // 8 digit number
         FindCommand expectedFindCommand =
-                new FindCommand(List.of(new PhoneMatchesNumberPredicate("87654321")));
+                new FindCommand(List.of(new PhoneContainsNumberPredicate("87654321")));
```
Comment (cowlinn) [inline src/test/java/longtimenosee/logic/parser/FindCommandParserTest.java:74]: Just a minor nitpick, maybe we could test for something closer to the max boundary of 25 digit numbers (maybe a 24 digit number?)

**Review by cowlinn — COMMENTED** [review]
LGTM!
